extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/JpaOperations.java

```diff
@@ -68,7 +69,11 @@ public static void flush() {
     // Private stuff
 
     public static EntityManager getEntityManager() {
-        return Arc.container().instance(EntityManager.class).get();
+        EntityManager entityManager = Arc.container().instance(EntityManager.class).get();
+        if (entityManager == null) {
```

Comment:
I would use `isAvailable()` here instead of doing the `get()` directly.

It makes things more readable IMHO.